WalletWasabi.Fluent/Controls/TagsBox.axaml.cs

```diff
@@ -30,7 +30,6 @@ public class TagsBox : TemplatedControl
 	private IList<string>? _suggestions;
 	private IEnumerable<string>? _items;
 	private IEnumerable<string>? _topItems;
-	private bool _requestAdd;
```

Comment:
I merged https://github.com/zkSNACKs/WalletWasabi/pull/10832 so this field is needed.

Reply:
ok. do you know why we don't see a conflict here then?

Reply:
Previously a logic has been changed to fix an issue, and this field is leftover from there. But that change re-introduced an old bug.
So https://github.com/zkSNACKs/WalletWasabi/pull/10832 restores that logic + fixes the bug that the modification was meant to fix.

Long story short, it is ok not to have a conflict, because https://github.com/zkSNACKs/WalletWasabi/pull/10832 doesn't touch this field.

Reply:
I added it back. 69036a815e5afcaa057af797bd13ec3ae95dda59